docs/src/content/docs/ops/backup-and-restore/overview.adoc

```diff
@@ -0,0 +1,58 @@
+---
+title: Backup and Restore
+---
+
+This document provides a high-level overview of XTDB’s stateful architecture and outlines what data is stored where, what typical backups include, and how to think about restoring XTDB in different failure scenarios.
+
+== Overview of Stateful Components
+
+XTDB consists of two primary stateful components:
+
+=== Log
+
+A totally ordered, append-only log.
+
+* This contains all transactions (representing one or more record-level operations) and their metadata, and inter-node messages. 
+* Guarantees ACID-compliant serial execution of all writes.
+* Pluggable implementations include:
+** *In-memory* (ephemeral - any unindexed data is lost on restart)
+** *Local disk* (durable on a single node)
+** *Remote/Kafka* (distributed, durable across nodes)
+* Every transaction is recorded immutably and linked to a durable log offset.
+
+=== Storage Module
+
+A immutable store containing the data and indexes that make up the database.
+
+* Structured as https://arrow.apache.org[Apache Arrow^] columnar files for performance and flexibility.
+* Data is periodically compacted into increasingly efficient layers.
+** *Level-0* files contain full snapshots of each table.
```

Comment:
(weak opinion) I'm not sure this (details of how compaction works) is what a reader reading this page would necessarily care about...? :thinking: from the PoV of the storage, it doesn't know nor need to care about what files are going in there, maybe only that these are snapshots of the log.